docker-compose.yml

```diff
@@ -74,10 +74,7 @@ services:
       - ./docker/nginx/addons.conf:/etc/nginx/conf.d/addons.conf
       - ./static:/srv/static
       - ./site-static:/srv/site-static
-      - storage:/shared_storage/uploads:/srv/user-media
-      - storage:/files:/srv/user-media/addons
-      - storage:/guarded-addons:/srv/user-media/guarded-addons
-      - storage:/sitemaps:/srv/user-media/sitemaps
+      - storage:/srv/user-media
```

Comment:
I wonder if it is possible to test this. `./.github/actions/run-docker` has nginx running but not the `/etc/host` so would have to use localhost I would guess.

Reply:
Can we predict a file URL that goes through all of this?  In the meantime you'll need this to be able to complete #22408 so I suppose manual confirmation would be a good first step.

Reply:
I think we could populate a file in an expected path in ./storage and then verify that a call to nginx returns the file. We don't have to add it here, but can you file a follow up ticket to do so. This was broken for a while and no one noticed.

Reply:
Yep, I filed https://github.com/mozilla/addons/issues/14886